docs/examples/dialog_box.md

```diff
@@ -68,5 +68,34 @@ def test_num_input():
     )
     response = hardpy.run_dialog_box(dbx)
     assert response == 123.0
+
+
+@pytest.mark.case_name("Radiobutton")
+def test_radiobutton():
+    dbx = DialogBox(
+        dialog_text=f"Select one item (1) from several and click the Confirm button",
```

Comment:
`Select item "one" out of several and click Confirm.`

Reply:
Ready